incubator/bastion/templates/deployment.yaml

```diff
@@ -0,0 +1,125 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+spec:
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+      annotations:
+        pod.beta.kubernetes.io/init-containers: '[{
+          "name": "init",
+          "image": "{{ .Values.image.repository }}:{{ .Values.image.tag }}",
+          "imagePullPolicy": {{ .Values.image.pullPolicy | quote }},
+          "command": ["/bin/sh", "-c", "(cp -a /etc/ /mnt/etc/) && (mkdir -p /mnt/home /mnt/root /mnt/var/log)"],
```

Comment:
oh wait... why are we modifying `incubator/bastion/templates/deployment.yaml` in this PR? =)